<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wally::qtfront::question_view_sub::ChoiceMimic</class>
 <widget class="QGroupBox" name="wally::qtfront::question_view_sub::ChoiceMimic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Choice</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ChoiceTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_moveUp">
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_moveDown">
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChoiceTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qtfront/question_view_sub/choice_mimic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
